.github/workflows/generator-draft-release.yaml

```diff
@@ -17,3 +34,6 @@ jobs:
         run: npx semantic-release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Clean release tags
+        if: always()
+        run: ./clean-release-tags.sh
```

Comment:
We can run this step at the start of the workflow to cleanup any previous mistakenly-created tags. What are your thoughts?

Reply:
Do it!

Reply:
Done ✅